products/couchbase-server.md

```diff
@@ -31,8 +31,8 @@ releases:
     releaseDate: 2021-08-31
 -   releaseCycle: "6.6"
     eol: 2023-01-01
-    latest: "6.6.5"
-    latestReleaseDate: 2022-12-09
+    latest: "6.6.6"
```

Comment:
6.x should be EOL according to the description (2023-01-01). May it should be updated ?

Reply:
Might be worth reaching out to them to confirm https://www.couchbase.com/contact/

Reply:
I may have an explanation, see #2375.